src/components/GoalForm.js

```diff
@@ -105,7 +105,7 @@ class GoalForm extends Component {
         timestamp_created: moment().unix(),
         hierarchy: 'Branch',
         status: this.props.status,
-        description: '',
```

Comment:
oh, this is a bug fix! good catch. Can you also fix 'hierarchy'

Reply:
finished

Reply:
remove dist/img/squirrel icon.zip : finished *

we want to also enable the history to show the adding/removing of 'members' to the Goal, in the activity history. These are to be found in the 'members' property of the goalHistory  : Finished